cockatrice/src/window_main.cpp

```diff
@@ -285,7 +285,7 @@ void MainWindow::actAbout()
 {
     QMessageBox mb(QMessageBox::NoIcon, tr("About Cockatrice"), QString(
         "<font size=\"8\"><b>Cockatrice</b></font><br>"
-        + tr("Version %1").arg(VERSION_STRING)
+        + tr("Version") + QString(" %1").arg(VERSION_STRING)
```

Comment:
That would miss a space and require an implicit cast from a const char* to QString.